canonical-web-and-design/jaas-dashboard
# Sharing styling tweaks

## Done

- Fix display of the 'time since' value.
- Fix the styling of a number of elements to match the design.

## QA

- View a model with shared users to ensure that the UI closer matches the design.
https://app.zeplin.io/project/60784aaae76066037acf386a/screen/607ed092c6f95397514e028d

## Details

Fixes https://github.com/canonical-web-and-design/app-squad/issues/150

## Screenshots

![Screen Shot 2021-07-07 at 5 04 32 PM](https://user-images.githubusercontent.com/532033/124839115-6c70c780-df45-11eb-82b9-a6fe49109251.png)
